### DISCUSSION TITLE
High quality DAC interfacing

### DISCUSSION THREAD

**Discussion by SandeepJagdev** [pschatzmann/arduino-audiokit#103] [Q&A] [answered]
I want to interface PCM5242 (DAC by Texas Instrument) with ESP32.
Is possible to run it with full efficiency(32-bit, 384KHz)??

**Comment by pschatzmann** [comment, marked as answer]
I don't think this is the proper place to ask this question: After all ADF does not support PCM5242.
So you would need to add this yourself.

I also don't know what the max supported I2S speed is for the ESP32. I guess you will need to look this up yourself in the technical sheet.
